server/app.js

```diff
@@ -60,5 +61,5 @@ app.use("/resources", resourceRouter);
 app.use("/job-applications", jobApplicationsRouter);
 app.use("/posts/jobs", postsJobsRouter);
 app.use("/companies", companiesRouter);
-
+app.use("/profiles", profilesRouter);
```

Comment:
We don't need extra routers to render these views. as we don't want these views available unless your logged in and it would take more code than necessary  to prevent that.

Reply:
He may have came across a use case where it was better to have it in it's own route. It looks like there isn't a getBootcampUserById route established prior to this PR but there was one for companies (getCompanyById) in the companies.js route.